edkahara/iReporter
# [finishes #162361831)] clean up code

__What does this PR do?__
Clean up the project's code.

__Description of tasks to be completed.__
Replace any redundant api endpoint, unit test and .travis.yml code.

__How should it be manually tested?__
Clone the project. Check out the ch-clean-up-code#162361381 branch. Start the environment using env\Scripts\activate. Run flask run on the terminal and go to Postman. To perform the tests run nosetest.

__Relevant pivotal tracker stories.__
#162361381
